.gitignore

```diff
@@ -1,5 +1,7 @@
-vendor/
-report/*
-Resources/doc/_build/
+/build
+/Resources/doc/_build
+/vendor
+/report/*
```

Comment:
Not sure this is still needed (I see no reference to it in the project). @dbu @lsmith77 ?